modules/client/src/main/ts/api/UnitTest.ts

```diff
@@ -115,3 +125,9 @@ export const test = (name: string, test: () => void) => {
     }
   });
 };
+
+/** Tests an async function (function that returns a Promise). */
+export const promiseTest = (name: string, test: () => Promise<void>) =>
+  asyncTest(name, (success, failure) => {
+    test().then(success).catch(failure);
```

Comment:
We've had this in the past, I'm not sure why it was removed. Polyfill required on IE11, anyway.

Reply:
As discussed out-of-band, we'll leave it up to the caller to polyfill Promises.

Reply:
That's fine for tests _using_ bedrock, but not bedrock's tests.